valhalla/baldr/graphtileheader.h

```diff
@@ -165,21 +165,16 @@ class GraphTileHeader {
    * @return Returns the base lat,lon of the tile (degrees).
    */
   midgard::PointLL base_ll() const {
-    GraphId id(graphid_);
-
-    if (id.level() == TileHierarchy::GetTransitLevel().level) {
-      return TileHierarchy::GetTransitLevel().tiles.Base(id.tileid());
-    }
-    return TileHierarchy::levels()[id.level()].tiles.Base(id.tileid());
+    return {base_ll_[0], base_ll_[1]};
```

Comment:
this might make https://github.com/valhalla/valhalla/pull/5531 almost redundant, but let's keep it anyway, a bit less pointll construction:) cc @SiarheiFedartsou